compose/ui/ui-uikit/src/uikitMain/objc/CMPUIKitUtils/CMPUIKitUtils/CMPGestureRecognizer.h

```diff
@@ -9,21 +9,24 @@
 #import <UIKit/UIGestureRecognizerSubclass.h>
 
 NS_ASSUME_NONNULL_BEGIN
-
+ 
 @protocol CMPGestureRecognizerHandler <NSObject>
-
+ 
```

Comment:
Sry, it looks like here some formatting issues with spaces.

Reply:
Fixed